src/client/pages/OfferNew/Checkout/SignStatus.tsx

```diff
@@ -1,75 +1,132 @@
 import styled from '@emotion/styled'
-import React from 'react'
+import React, { useEffect, useState } from 'react'
 import { motion } from 'framer-motion'
 import { colorsV3 } from '@hedviginsurance/brand/dist'
-import { SignState, SignStatus as GraphQLSignStatus } from 'data/graphql'
+import { SignStatus as GraphQLSignStatus, BankIdStatus } from 'data/graphql'
 import { useTextKeys } from 'utils/textKeys'
+import { WarningIcon } from '../../../components/icons/Warning'
 import { SignUiState } from './Sign'
+import { LinkToChat } from './LinkToChat'
+
+type Props = {
+  signStatus: GraphQLSignStatus | null
+  signUiState: SignUiState
+  isLoading: boolean
+}
+
+enum StatusMessageTextKey {
+  REDIRECTING = 'CHECKOUT_SIGN_STARTED_WITH_REDIRECT',
+  GENERIC_ERR = 'CHECKOUT_SIGN_GENERIC_ERROR',
+  START_SIGNING = 'SE_BANKID_START_SIGN',
+  CERTIFICATE_ERR = 'SE_BANKID_CERTIFICATE_ERROR',
```

Comment:
this is nit but i'm not sure i like shortening it to `err`. Err is i guess quite well accepted but i see no real reason here so we could just write it out imo

Reply:
I agree, I was just annoyed when selecting all instances of `CERTIFICATE_ERROR` I also got that part of the `SE_BANKID_CERTIFICATE_ERROR` 😄 So I just wanted to differentiate them in some way, but I actually don't like `err` either. Can change it!